MapboxVisionAR/ViewController/LaneVisualParams.swift

```diff
@@ -0,0 +1,21 @@
+/**
+ Represents parameters to configure look and feel of AR lane.
+ */
+public struct LaneVisualParams {
+    /// Color of the lane represented in RGBA. It takes alpha channel into account.
+    public var color = UIColor(red: CGFloat(ARConstants.laneDefaultColor.x),
+                        green: CGFloat(ARConstants.laneDefaultColor.y),
+                        blue: CGFloat(ARConstants.laneDefaultColor.z),
+                        alpha: CGFloat(ARConstants.laneDefaultColor.w))
+    /// Width of the lane.
```

Comment:
We should probably specify that it's measured in meters.

Reply:
Done, thanks!